networking_cisco/plugins/cisco/cfg_agent/device_drivers/asr1k/aci_asr1k_routing_driver.py

```diff
@@ -401,18 +471,19 @@ def _set_snat_pools_from_hosting_info(self, ri, gw_port, is_delete):
             pool_name = "%s_nat_pool" % (vrf_name)
             self._do_set_snat_pool(pool_name, subnet['ip'],
                                    subnet['ip'], str(net.netmask), is_delete)
-            secondary_ip = netaddr.IPAddress(net.value +
-                                             (net.hostmask.value - 1))
-            if is_delete:
-                self._asr_do_remove_secondary_ip(secondary_ip,
-                                                 gw_port, str(net.netmask))
-            else:
-                self._asr_do_add_secondary_ip(secondary_ip,
-                                              gw_port, str(net.netmask))
+            # TODO(tbachman): re-enable, once fixed
```

Comment:
It means that the secondary IPs aren't being configured, so the ASR won't respond to ARPs on that subnet. This is going to be addressed in a subsequent patch.